### PR TITLE
Issue #13213:Removed '//ok' from javadocparagraph

### PR DESCRIPTION
**Pull request by 555vedant**
"Issue #13213:Removed '//ok' from javadocparagraph"

### CODE REVIEW

**Comment by 555vedant** [comment]
Screenshot 2024-02-25 112045

File: src/test/resources/com/puppycrawl/tools/checkstyle/checks/javadoc/javadocparagraph/InputJavadocParagraphCorrect.java [hunk 1]
```diff
@@ -9,15 +9,15 @@
 package com.puppycrawl.tools.checkstyle.checks.javadoc.javadocparagraph;
 
 /**
- * Some Javadoc. // ok
```
Comment (MANISH-K-07) [inline src/test/resources/com/puppycrawl/tools/checkstyle/checks/javadoc/javadocparagraph/InputJavadocParagraphCorrect.java:12]: Please remove the trailing whitespace here.

File: src/test/resources/com/puppycrawl/tools/checkstyle/checks/javadoc/javadocparagraph/InputJavadocParagraphCorrect.java [hunk 2]
```diff
@@ -9,15 +9,15 @@
...
+ * Some Javadoc. 
  *
  * <p>Some Javadoc.
  *
  */
 class InputJavadocParagraphCorrect {
 
     /**
-     * Some Javadoc. // ok
```
Comment (MANISH-K-07) [inline src/test/resources/com/puppycrawl/tools/checkstyle/checks/javadoc/javadocparagraph/InputJavadocParagraphCorrect.java:20]: Same as above....

**Review by MANISH-K-07 — COMMENTED** [review]
Please restrict all your changes to a single commit.
Squash and rebase if necessary

Items :

**Review by abhishekmaity — APPROVED** [review]
Looks as required.

File: src/test/resources/com/puppycrawl/tools/checkstyle/checks/javadoc/javadocparagraph/InputJavadocParagraphCorrect.java (hunk 1 above)
Comment (abhishekmaity) [reply src/test/resources/com/puppycrawl/tools/checkstyle/checks/javadoc/javadocparagraph/InputJavadocParagraphCorrect.java:12]: @MANISH-K-07 place a code 'request change' instead would more useful in review. The contributor can directly apply the changes then.

**Comment by abhishekmaity** [comment]
> "Issue #13213:Removed '//ok' from javadocparagraph"

You can also add images/screen-shots via attachments using 'paper-clip' icon at the right side of menu in description/comment box

File: src/test/resources/com/puppycrawl/tools/checkstyle/checks/javadoc/javadocparagraph/InputJavadocParagraphCorrect.java (hunk 1 above)
Comment (555vedant) [reply src/test/resources/com/puppycrawl/tools/checkstyle/checks/javadoc/javadocparagraph/InputJavadocParagraphCorrect.java:12]: @abhishekmaity  yes sir it definitely helps lot !!

**Comment by 555vedant** [comment]
> Please resolve conflicts with CI. Looks like you need to restrict all your changes to a single commit. Squash and rebase if necessary
> 
> Items :

 sir , is this necessary ? this will be final change after resolving  ?

**Comment by MANISH-K-07** [comment]
> sir , is this necessary ? this will be final change after resolving ?

@555vedant , Please avoid addressing as "sir"

Checkstyle rules require all the changes in a single commit.
You need to verify the build using `mvn clean verify` before pushing up. The final change is when the build is a success on local

**Comment by 555vedant** [comment]
> > sir , is this necessary ? this will be final change after resolving ?
> 
> @555vedant , Please avoid addressing as "sir"
> 
> Checkstyle rules require all the changes in a single commit. You need to verify the build using `mvn clean verify` before pushing up. The final change is when the build is a success on local

Screenshot 2024-02-25 111726
this is not fine ?

**Comment by 555vedant** [comment]
> @555vedant , Snapshots as above aren't required. Green CI is enough.
> 
> Please resolve conflicts with CI. Looks like you need to restrict all your changes to a single commit. Squash and rebase if necessary
> 
> Items :

i am working on it !

**Review by nrmancuso — CHANGES_REQUESTED** [review]
@555vedant please revert all changes that are not simple `// ok` comment removal. Test inputs are intentionally formatted incorrectly/ bad code/ etc.

**Comment by 555vedant** [comment]
sorry sir i am working hard on it but still but i will fix it

**Comment by 555vedant** [comment]
i will make new PR for this issue because i have lots of modifcation in my forked repo

**Pull request closed without merging**